tests/api/test_patterns.py

```diff
@@ -67,14 +114,117 @@ def test_patterns_quarterly_one_level(holding_lib_martigny_w_patterns):
     assert holding.next_issue_display_text == 'no 73 mars 2023'
     # test preview
     issues = holding.prediction_issues_preview(13)
-    assert issues[-1] == 'no 85 mars 2026'
+    assert issues[-1]['issue'] == 'no 85 mars 2026'
+
+
+def test_receive_regular_issue(holding_lib_martigny_w_patterns):
```

Comment:
I think theses two tests should not be in the same file/directory one in tests/api and the other tests/ui